sageintacctsdk/apis/ar_payments.py

```diff
@@ -0,0 +1,29 @@
+"""
+Sage Intacct AR Payments
+"""
+from typing import Dict
+
+from .api_base import ApiBase
+
+
+class ARPayments(ApiBase):
+    """Class for AR Payments APIs."""
+    def __init__(self):
+        ApiBase.__init__(self, dimension='ARPYMT')
+
+    def get_all(self):
```

Comment:
We have a get_all() available already in the ApiBase class, so this is not needed. You can also make use of read_by_query() from the base class if required. Rest all changes looks fine.

Reply:
Thank you, I will make use of the get_all() method in ApiBase and remove this method from the changes in this PR